### PR TITLE
Remove the reminder endpoint from this repo and add it to api-gateway

### PR DESCRIPTION
**Pull request by jlieb10**
## Why are you doing this?
We need to access the reminder api from www.theguardian.com & support.theguardian.com -- and decided the best place for the endpoint to be was not support.theguardian.com - it should be a more general location, so API gateway made sense

[**Trello Card**](https://trello.com/c/tv5Q7zXa/1731-remove-reminders-endpoint-from-support-frontend-and-implement-with-api-gateway)

## Changes

* Use the new api endpoint in this repo
* remove the old, now unused route, service, controller, etc.

### CODE REVIEW

File: support-frontend/assets/helpers/routes.js [hunk 1]
```diff
@@ -38,9 +38,12 @@ const routes: {
   guardianWeeklySubscriptionLandingGift: '/subscribe/weekly/gift',
   postcodeLookup: '/postcode-lookup',
   createSignInUrl: '/identity/signin-url',
-  createReminder: '/contribute/remind-me',
+  createReminder: 'https://contribution-reminders-code.support.guardianapis.com/remind-me',
```
Comment (paulbrown1982) [inline support-frontend/assets/helpers/routes.js]: Will this still work @jlieb10?

Comment (jlieb10) [reply support-frontend/assets/helpers/routes.js]: No that needs to be cleaned up - thank you

Comment (jlieb10) [reply support-frontend/assets/helpers/routes.js]: using `createReminderEndpoint` (below) instead

Comment (jlieb10) [reply support-frontend/assets/helpers/routes.js]: (which should probably be `getReminderEndpoint` actually)

Comment (jlieb10) [inline support-frontend/assets/helpers/routes.js]:
```suggestion
```

**Pull request merged**